content/microservices/frontend/tabs/cdk.md

```diff
@@ -272,7 +272,7 @@ cdk deploy --require-approval never
 
 ```bash
 alb_url=$(aws cloudformation describe-stacks --stack-name ecsworkshop-frontend --query "Stacks" --output json | jq -r '.[].Outputs[] | select(.OutputKey |contains("LoadBalancer")) | .OutputValue')
-siege -c 20 -i $alb_url&
+siege -c 20 -i $alb_url
```

Comment:
The background option, especially because the `&` has no space after `$alb_url` would lead to confusion. Ultimately running the command in the background is unnecessary since your goal is to generate load and then scale appropriately.

Reply:
I have updated `cdk.md` with that change.